### PR TITLE
EPMLDI-1708

### PR DESCRIPTION
**Pull request by lab-lamz4**
Set tags to the volume created by molecule

### CODE REVIEW

File: AWS/create.yml [hunk 1]
```diff
@@ -59,6 +59,38 @@
...
       retries: 300
 
+    - name: Get Volumes IDs
+      ec2_vol:
+        instance: '{{ item.instance_ids[0] }}'
+        state: 'list'
+      register: 'ec2_instances_volumes'
+      loop: '{{ instance_create.results }}'
+      loop_control:
+        label: 'ec2-{{ item.instance_ids[0] }}'
```
Comment (tgadiev) [inline AWS/create.yml]: Why do we need label if we're not using it here?

File: AWS/create.yml [hunk 2]
```diff
@@ -59,6 +59,38 @@
...
+
+    - name: 'Set facts to tag Volumes'
+      set_fact:
```
Comment (tgadiev) [inline AWS/create.yml]: Why do we need that step?

File: AWS/create.yml (hunk 1 above)
Comment (lab-lamz4) [reply AWS/create.yml]: Fixed

File: AWS/create.yml (hunk 2 above)
Comment (lab-lamz4) [reply AWS/create.yml]: Fixed

**Review by pavelpikta — APPROVED** [review]

**Review by tgadiev — CHANGES_REQUESTED** [review]
How about WINRM volume tags?

**Review by tgadiev — APPROVED** [review]

**Review by tgadiev — APPROVED** [review]

**Pull request merged**